example.env

```diff
@@ -0,0 +1,3 @@
+OPENAI_API_KEY=""
+SUPABASE_URL=""
+SUPABASE_ANON_KEY=""
```

Comment:
Do you mind naming `.env.example` please? This way it always stays close to the `.env.local` or other files